src/components/testCaseLanding/qdm/CreateCodeCoverageNavTabs.tsx

```diff
@@ -50,9 +58,27 @@ export default function CreateCodeCoverageNavTabs(props: NavTabProps) {
     testCasePassFailStats,
     coveragePercentage,
     validTestCases,
+    selectedPopCriteria,
   } = props;
 
   const featureFlags = useFeatureFlags();
+
+  const shouldDisableRunTestsButton = () => {
```

Comment:
Here this is decided based on selected population criteria. This may not be correct as we execute the whole measure and not a single criteria/group. however, in the edit test case, it is based on at least one population criteria/group of a measure which I think is correct.

Reply:
@adongare code was reverted, thanks!